### PR TITLE
added a h1

### PR DESCRIPTION
**Pull request by soshochang**

### CODE REVIEW

File: frontPage [hunk 1]
```diff
@@ -5,5 +5,6 @@
 </head>
 <body>
   <h1>Hello!</h1>
+  <h1>Hello!!</h1>
```
Comment (dancinturtle) [inline frontPage:8]: You just copied my h1. That's silly.

**Review by dancinturtle — COMMENTED** [review]
Hi! I like this review feature, I think.

File: frontPage (hunk 1 above)
Comment (soshochang) [reply frontPage:8]: yes i did

Comment (dancinturtle) [reply frontPage:8]: Is this a chat box?

**Pull request merged**